eyre/src/lib.rs

```diff
@@ -320,6 +320,7 @@
     feature(rustdoc_missing_doc_code_examples),
     warn(rustdoc::missing_doc_code_examples)
 )]
+#![cfg_attr(not(nightly_features), warn(private_in_public))]
```

Comment:
We need to wait for the next beta as per the warnings you can see below.

Reply:
What warnings are you referring to?

Reply:
If you check the diff, you can see github comments of clippy warnings in `beta`.

Reply:
Oh! Thank you for pointing that out, I wasn't looking in the right place : ) I'll expand the nightly check to also check for the beta toolchain here.

Do you think it'd be preferable to factor that part of the build script into a library or to keep copying it over?

Also, is there a way you trigger halfway comprehensive CI locally, or do you trigger it on your fork or something?